gtsfm/two_view_estimator_cacher.py

```diff
@@ -0,0 +1,106 @@
+"""Decorator which implements a cache for the two-view-estimator class
+
+Authors: Ayush Baid
+"""
```

Comment:
should this file go under gtsfm/frontend/cacher/two_view_estimator_cacher? @ayushbaid

Reply:
Yes good idea.